admin/services/data-access/retry-async.js

```diff
@@ -16,10 +16,11 @@ const defaultConfiguration = {
  */
 const asyncRetryHandler = async (asyncRetryableFunction, retryConfiguration = defaultConfiguration, retryCondition = defaultRetryCondition) => {
   const retryPolicy = {}
+  let result
   try {
     Object.assign(retryPolicy, retryConfiguration)
     logger.debug('asyncRetryHandler: executing retryable method...')
-    const result = await asyncRetryableFunction()
```

Comment:
I don't see the need to turn the `const` into a `let`.  It's block scoped.

Reply:
i assign `result` further down, hence the `let`

Reply:
It's in a different block, so could also be const I think.